### PR TITLE
[PPML] Update trusted-machine-learning readme

### PR DESCRIPTION
**Pull request by SEULiyao**
## Description

Added a description of submitting a machine learning job with k8s

### CODE REVIEW

File: ppml/trusted-machine-learning/README.md [hunk 1]
```diff
@@ -114,6 +114,60 @@ export your_application_arguments=...
     ${your_application_arguments}
 ```
 
+#### 2.3 Submit Spark Machine Learning job with k8s
```
Comment (Uxito-Ada) [inline ppml/trusted-machine-learning/README.md:117]: Kubernetes not k8s in this doc

File: ppml/trusted-machine-learning/README.md [hunk 2]
```diff
@@ -114,6 +114,60 @@ export your_application_arguments=...
...
+You can also use bigdl-ppml-submit.sh to submit Machine Learning job with k8s.
+```bash
+export secure_password=`openssl rsautl -inkey /ppml/password/key.txt -decrypt </ppml/password/output.bin`
+
+bash bigdl-ppml-submit.sh \
```
Comment (Uxito-Ada) [inline ppml/trusted-machine-learning/README.md:122]: Pls move the commands into a script and only bash the script in the doc

**Comment by Uxito-Ada** [comment]
Pls rename section 2.2 to a local one

File: ppml/trusted-machine-learning/README.md [hunk 3]
```diff
@@ -114,6 +114,60 @@ export your_application_arguments=...
...
+    --conf spark.ssl.keyStore=/ppml/keys/keystore.jks \
+    --conf spark.ssl.keyStorePassword=$secure_password \
+    --conf spark.ssl.keyStoreType=JKS \
+    --conf spark.ssl.trustStore=/ppml/keys/keystore.jks \
+    --conf spark.ssl.trustStorePassword=$secure_password \
+    --conf spark.ssl.trustStoreType=JKS \
+    --class org.apache.spark.examples.ml.RandomForestClassifierExample \
+    --name RandomForestClassifierExample \
+    --verbose \
+    --jars local://${SPARK_HOME}/examples/jars/spark-examples_2.12-${SPARK_VERSION}.jar \
```
[the remark this answers is not in the document]
Comment (SEULiyao) [reply ppml/trusted-machine-learning/README.md:167]: the ml examples are from this jar, so it may be the version of spark mllib.

**Review by Uxito-Ada — APPROVED** [review]
lgtm

**Pull request merged**